src/connectors/ona-api/aggregateFormData.js

```diff
@@ -44,24 +44,32 @@ function processFormData(formData, indicatorField, aggregateOptions) {
       }
 
       return {
-        ...datum,
+        ...(datum.properties || datum),
         period: [year, month, weekMonth],
-        [submissionDateField]: datum[submissionDateField],
-        [groupByField]: datum[groupByField],
+        [submissionDateField]: (datum.properties || datum)[submissionDateField],
+        [groupByField]: (datum.properties || datum)[groupByField],
       };
     });
   } else if (aggregateOptions['date-parse']) {
-    data = data.map((datum) => {
+    data = (data.features || data).map((datum) => {
       const { split, chunk } = aggregateOptions['date-parse'];
       const datumDate = split
-        ? datum[submissionDateField].split(split)[chunk]
-        : datum[submissionDateField];
+        ? (datum.properties || datum)[submissionDateField].split(split)[chunk]
+        : (datum.properties || datum)[submissionDateField];
 
       return {
-        ...datum,
+        ...(datum.properties || datum),
         'period-date': new Date(datumDate),
       };
     });
+  } else {
+    data = (data.features || data).map((datum) => {
+      const datumDate = (datum.properties || datum)[submissionDateField];
+      return {
+        ...(datum.properties || datum),
+        period: datumDate,
+      }
+    });
   }
```

Comment:
@kahummer in this block it looks like we're still aggregating by some sort of date: `submissionDateField`.

Should / could we add a line to determine which datum property should be use for `period`?